config.js

```diff
@@ -1,18 +1,36 @@
-export const API_URL =
-  process.env.NODE_ENV === 'production'
-    ? 'https://api.core.ac.uk/internal'
-    : 'https://api.dev.core.ac.uk/internal'
+const NODE_ENV = process.env.NODE_ENV || 'development'
 
-export const getLoginPage = (path = window.location.origin) => {
-  if (process.env.NODE_ENV === 'production') {
-    return new URL(
-      `/login.html?continue=${encodeURIComponent(path)}`,
-      window.location.origin
-    ).toString()
-  }
+const local = {
+  API_URL: 'http://127.0.0.1:8000/internal',
+  IDP_URL: 'http://127.0.0.1:8000',
+  LOGIN_URL: 'http://127.0.0.1:8000/test/login',
+  LOGOUT_URL: 'http://127.0.0.1:8000/logout',
```

Comment:
@Joozty are not `LOGIN_URL` and `LOGOUT_URL` obsolete?

Reply:
Yep they can be removed

Reply:
Is it automatically discovered from IDP URL? Could you please point me where I can find how it's actually work?

Reply:
The only thing you should change is [here](https://github.com/oacore/dashboard/blob/master/pages/_app/index.jsx#L208) For the logout redirection param should look like `continue={window.location.origin}?reason=logout`.

Reply:
It's not so according to [the implementation](https://github.com/oacore/dashboard/pull/198/files#diff-ac9542b946ce084e7a1b3976bd207257R84). It may be either my fault or artefact from #186.

Could you please double check it and help me to resolve this. What direction should I go?

Reply:
Simply just remove all changes you did in logout.jsx. And replace API_URL with IDF_URL. Obviously you can extract `LOGOUT_URL` into a function but the logic should stay the same.

Overall it should look like

```
import React from 'react'
import { AppBar } from '@oacore/design'

import { IDP_URL } from 'config'
import styles from './logout.css'

import { withGlobalStore } from 'store'

const LOGOUT_URL = `${IDP_URL}/logout?continue=${encodeURIComponent(
  typeof window !== 'undefined'
    ? `${window.location.origin}/?reason=logout`
    : ''
)}`

const Logout = ({ store }) => {
  if (!store.user) return null
  const name = store.user.email

  return (
    <AppBar.Item className={styles.container}>
      <span className={styles.name}>{name}</span>{' '}
      <a href={LOGOUT_URL}>Logout</a>
    </AppBar.Item>
  )
}

export default withGlobalStore(Logout)
```

Does that make sense?

Reply:
Thank you!